### PR TITLE
Add ZotHacks Open Graph Properties

### PR DESCRIPTION
**Pull request by samderanova**
Graphics has provided us with an image we can use for the [Open Graph protocol](https://ogp.me/). I've placed the image in the `public/` directory of the site and added some OG properties and values in `layout.tsx`.

### CODE REVIEW

**Comment by alexanderl19** [comment]
https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image

**Review by alexanderl19 — CHANGES_REQUESTED** [review]
https://nextjs.org/docs/app/api-reference/file-conventions/metadata/opengraph-image

File: apps/site/src/app/layout.tsx [hunk 1]
```diff
@@ -9,6 +9,10 @@ import "@/lib/styles/globals.scss";
 export const metadata: Metadata = {
 	title: "ZotHacks 2023",
 	description: "Hack at UCI's premier hackathon for beginners at UCI",
+	openGraph: {
+		title: "ZotHacks 2023",
```
Comment (alexanderl19) [inline apps/site/src/app/layout.tsx:13]: Should `openGraph.description` be set to the same value?

Comment (samderanova) [reply apps/site/src/app/layout.tsx:13]: Sure, we can set it to be the same as `metadata.description`.

**Review by alexanderl19 — APPROVED** [review]

**Pull request merged**